src/libsync/propagateremotedeleteencrypted.h

```diff
@@ -24,7 +24,7 @@ class PropagateRemoteDeleteEncrypted : public AbstractPropagateRemoteDeleteEncry
 public:
     PropagateRemoteDeleteEncrypted(OwncloudPropagator *propagator, SyncFileItemPtr item, QObject *parent);
 
-    virtual void start() Q_DECL_OVERRIDE;
+    void start() Q_DECL_OVERRIDE;
```

Comment:
@mgallien Don't we want to change `Q_DECL_OVERRIDE` to just `override`, for consistency?

Reply:
yes good point
I will do it